client/landing/gutenboarding/available-designs.ts

```diff
@@ -9,12 +9,15 @@ import { addQueryArgs } from '@wordpress/url';
 import { isEnabled } from '../../config';
 import type { Design } from './stores/onboard/types';
 const availableDesignsConfig = require( './available-designs-config.json' );
+const availableFSEDesignsConfig = require( './available-fse-designs-config.json' );
 
 interface AvailableDesigns {
 	featured: Design[];
 }
 
-const availableDesigns: Readonly< AvailableDesigns > = availableDesignsConfig;
+const availableDesigns: Readonly< AvailableDesigns > = isEnabled( 'gutenboarding/site-editor' )
```

Comment:
I agree this is probably best direction.

In intermediate, to me it seems totally fine to also just override `theme` value in _all_ designs when reading the config. That would be very little code changes until we know more, and until we have this coming from an API anyway.

You could switch Gutenboarding to FSE mode with a URL like `wpcalypso.wordpress.com/new?flags=gutenboarding/site-editor` and everything would look as before, but theme would end up being different and final redirection URL would be different.

No strong opinion on this tho and `is_fse` seems fine too  (or opposite with `is_legacy`? Everything will be FSE eventually.).

To me separate config file is least preferable at this point, as that seems like a maintenance trouble.

Reply:
> it seems totally fine to also just override theme value in all designs when reading the config

I think this will cause issues with design preview if I'm not mistaken. (Since the screenshots would not be generated via that script)

Reply:
these changes are done now, other than integrating with `is_edge`

Reply:
> I think this will cause issues with design preview if I'm not mistaken. (Since the screenshots would not be generated via that script)

You'd generate another set of preview thumbnails with `gutenboarding/site-editor` flag active locally; you'd end up with different set of files where unique URL has the new theme slug and they'd work just fine alongside other thumbnail files.

Reply:
Ah good idea. I'm not sure if we should go this extra step... I wonder if it would be confusing to switch out all of the themes if the flag is active?

Either way, it could be easy to add.

Currently, with the fag active:
- site editor is enabled
- gutenberg edge is enabled
- redirect to site editor
- theme is matched to whatever design you chose

Currently, without the flag, you can choose an FSE design and all of the above happens, plus the theme is set to seedlet-blocks.

I could see why overriding all designs with the flag might be helpful for now, I'll look into that

Reply:
@simison This is done now. I refactored `getAvailableDesigns` so that we could reuse it in the thumbnail script, and I also generated thumbnails for every design with seedlet-theme set.